.autover/changes/1916e292-d131-4c9a-8036-0d7b68f0ce7a.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Projects": [
+    {
+      "Name": "Amazon.Lambda.CognitoEvents",
+      "Type": "Patch",
+      "ChangelogMessages": [
+        "Make ForceAliasCreation nullable to allow proper serialization"
```

Comment:
Add `[Breaking Change]` as a prefix. It is a very minor breaking change but we should be as transparent as policy. Maybe rephrase the message as more of a bug fix then just stating a fact. Possible option.
```
[Breaking Change] Fixed issue not handling null value during deserialization for `ForceAliasCreation` by making it nullable.
```